ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -225,13 +229,16 @@ export default class ConfirmPageContainer extends Component {
               )}
             </PageContainerFooter>
           )}
-          {editingGas && (
-            <EditGasPopover
-              mode={EDIT_GAS_MODES.MODIFY_IN_PLACE}
-              onClose={handleCloseEditGas}
-              transaction={currentTransaction}
-            />
-          )}
+          {editingGas &&
+            (EIP_1559_V2 ? (
+              <AdvancedGasPopover onClose={handleCloseEditGas} />
+            ) : (
+              <EditGasPopover
+                mode={EDIT_GAS_MODES.MODIFY_IN_PLACE}
+                onClose={handleCloseEditGas}
+                transaction={currentTransaction}
+              />
+            ))}
```

Comment:
We need to be able to open advance gas modal from both transaction confirmation page and edit gas fee modal,
in addition back arrow on advance gas modal should work as expected.

Reply:
This was temporary till the edit gas modal was available. I am updating that now.

Reply:
cool